### DISCUSSION TITLE
Error: E_IOC_LOOKUP_FAILED

### DISCUSSION THREAD

**Discussion by sasha172017** [adonisjs/core#1820] [Help]
When i use model in my service which i add to container, i receive this error "E_IOC_LOOKUP_FAILED: Cannot resolve "Adonis/Lucid/Orm" namespace from the IoC Container", how it change?

**Comment by AddoSolutions** [comment]
Having this same issue

**Comment by douglasdnzcarvalho** [comment]
I was having this same issue, maybe the discussion https://github.com/adonisjs/core/discussions/1881 can help you.
